.mypy.ini

```
@@ -0,0 +1,95 @@
+[mypy]
+python_version = 3.7
+show_error_codes = True
+
+# ---------------- Needs better typing, project for a future pull -----
+
+[mypy-manim.__main__]
+ignore_errors = True
+
+[mypy-manim.renderer.*]
+ignore_errors = True
+
+[mypy-manim.camera.*]
+ignore_errors = True
+
+[mypy-manim.scene.*]
+ignore_errors = True
+
+[mypy-manim.cli.cfg.*]
+ignore_errors = True
+
+[mypy-manim.mobject.*]
+ignore_errors = True
+
+[mypy-manim._config.*]
+ignore_errors = True
+
+[mypy-manim.utils.*]
+ignore_errors = True
+
+[mypy-manim.animation.*]
+ignore_errors = True
```

```suggestion
[mypy-manim.__main__,manim.renderer.*,manim.camera.*,manim.scene.*,manim.cli.cfg.*,manim.mobject.*,manim._config.*,manim.utils.*,manim.animation.*]
ignore_errors = True
```

Why not do it like this?